### ISSUE TITLE
How to add range in v-mask?

### ISSUE THREAD

**Issue by VladyslavDP** [probil/v-mask#446]
### V-Mask and Vue versions
1.1.1

### Reproduction Link

https://jsfiddle.net/h3jf1ywu/

### What is Expected?
I need mask similar this regex - /\d{1,4}.\d{1,2}/gm
https://regex101.com/r/xlvFKi/1

### What is actually happening?
I need strongly follow declared mask
### Question
Is it possible to add range for digits before the point?
Here is an example with jquery.mask - http://jsfiddle.net/61g983df/

**Comment by greenkey-m** [comment]
As i think, its not possible in v-mask

**Comment by probil** [comment]
Yeah. It's not possible right now. But the next version will support passing array of regular expressions as a mask or function. Function is especially powerful as it can cover a lot of advanced use cases

**Comment by avxkim** [comment]
@probil by next version you mean 3.x or 2.3.0?

**Comment by probil** [comment]
I'm afraid passing an array of regular expressions won't help to achieve the desired format but functions  would probably do

@avxkim I ment 2.X, sot it's already there. Not sure text-mask _pipes_ are supported properly though 🤔
